Makefile

```diff
@@ -0,0 +1,34 @@
+PYTHON := python
+PIP := $(PYTHON) -m pip
+GRPC_TOOLS_VERSION := 1.62.1
+
+.PHONY: all update-deps generate-protos check
+
+all: update-deps generate-protos format
+
+update-deps:
+	$(PIP) install pip-tools
+	$(PYTHON) -m piptools compile requirements.in
+
+generate-protos: generate-and-prepend-preambles format
+
+format:
+	$(PIP) install pyink
+	pyink --pyink-indentation 2 --line-length 80 --verbose .
+
+check:
+	pylint --ignore-patterns=".*_pb2.py,.*_pb2_grpc.py" jetstream/ benchmarks/
+
```

Comment:
Thanks for adding this one! What is the benefit of it compared with current approach? It seems more complex.

Reply:
For adding the --ignore-patterns or for adding the target entirely? Added the --ignore-patterns flag so we dont need to add `pylint: disable=all` to files when we regenerate protos. Added the target entirely to make is usable elsewhere, for example if we want to add a precommit hook